modules/processors/nih_dataset_processor.py

```diff
@@ -0,0 +1,59 @@
+import datasets
+import os
+from ..dataset_processor import *
+
+class NIHDataset(Processor):
+
+    def __init__(self, config_name='number',is_query=True, *args, **kwargs):
+        self.dataset_name='nih_v1_'+config_name
+        if is_query:
+            self.dataset_name+='_query'
+        else:
+            self.dataset_name+='_doc'
+        self.conf_name= config_name
+        self.is_query=is_query
+        
+        super().__init__(*args, **kwargs, dataset_name=self.dataset_name)
+    
+    def process(self):
+        print(self.conf_name)
+        d = datasets.load_dataset('naver/bergen_nih_v1',self.conf_name)[self.split]
```

Comment:
I checked on my side and:
```
>>> dataset = datasets.load_dataset('naver/bergen_nih_v1', 'number')
dataset
>>> dataset
DatasetDict({
    train: Dataset({
        features: ['qid', 'query', 'label', 'did', 'doc'],
        num_rows: 512
    })
})
```
=> there is only a 'train' split on the hub.

Reply:
Same for 'simple' and 'multihop'

Reply:
By default, HF upload the dataset as train set.. Is this a big deal?